src/plugins/crypto/gpg.c

```diff
@@ -166,16 +197,39 @@ static int getGpgBinary (char ** gpgBin, KeySet * conf, Key * errorKey)
 	}
 
 
-	// last resort
-	const size_t defaultLen = strlen (ELEKTRA_CRYPTO_DEFAULT_GPG_BIN) + 1;
-	*gpgBin = elektraMalloc (defaultLen);
+	// last resort number one - check for gpg2 at /usr/bin/gpg2
+	// NOTE this might happen if the PATH variable is empty
+	*gpgBin = strdup (ELEKTRA_CRYPTO_DEFAULT_GPG2_BIN);
```

Comment:
Please use elektraStrDup here.

And why use pointer to pointer? (not needed here, maybe you can put this fallback code to separate function altogether?)